### PR TITLE
OGCServer url can not be null, OGCServer name must be unique

### PR DESCRIPTION
**Pull request by ochriste**
fix https://github.com/camptocamp/c2cgeoportal/issues/2499

### CODE REVIEW

**Comment by sbrunner** [comment]
Thanks, that's gooks good :-)

File: c2cgeoportal/scripts/themev1tov2.py [hunk 1]
```diff
@@ -129,17 +130,24 @@ def ogc_server(session):
         if identifier not in unique_servers:
             unique_servers.append(identifier)
             new_ogc_server = OGCServer()
-            new_ogc_server.url = server[0]
+            url = server[0]
+            if url is None:
+                url = 'config://local/mapserv'
```
Comment (sbrunner) [inline c2cgeoportal/scripts/themev1tov2.py]: should be `config://internal/mapserv`

File: c2cgeoportal/scripts/themev1tov2.py [hunk 2]
```diff
@@ -154,8 +162,11 @@ def layer_v1tov2(session, layer):
         image_type = layer.image_type
         if layer.image_type is None:
             image_type = 'image/png'
+        url = layer.url
+        if layer.url is None:
+            url = 'config://local/mapserv'
```
Comment (sbrunner) [inline c2cgeoportal/scripts/themev1tov2.py]: same

**Comment by ochriste** [comment]
corrected

**Comment by sbrunner** [comment]
the name error comes from that the line https://github.com/camptocamp/c2cgeoportal/pull/2500/files#diff-ca696626af2c24c1c56983a6b182e8beR129
is incoherent with:
https://github.com/camptocamp/c2cgeoportal/blob/dbd2ce8cc92be857d421451e212ad969f5172dcc/c2cgeoportal/scripts/themev1tov2.py#L147
...

**Comment by gberaudo** [comment]
LGTM

**Pull request merged**